packages/javascript/js-client-library/src/types.ts

```diff
@@ -360,6 +364,36 @@ export interface GetCollectorsResponse {
     };
 }
 
+export interface GetCommunityCollectorsResponse {
+    data: Record<CommunityCollectorType, CollectorManifest[]>;
+}
+
+export interface GetEnterpriseCollectorsResponse {
+    data: Record<EnterpriseCollectorType, CollectorManifest[]>;
+}
```

Comment:
These would be good to have in the adjacent responses.ts file! I might also update the type declaration to use the `BasicResponse` type for consistency:

```suggestion
export type GetCommunityCollectorsResponse = BasicResponse<Record<CommunityCollectorType, CollectorManifest[]>>;

export type GetEnterpriseCollectorsResponse = BasicResponse<Record<EnterpriseCollectorType, CollectorManifest[]>>;
```